packages/components/src/ListView/Items/Items.component.js

```diff
@@ -110,18 +110,18 @@ export class ItemsComponent extends React.PureComponent {
 
 	renderToggleAll() {
 		const { id, isSwitchBox, toggleAllChecked, onToggleAll, t } = this.props;
+		const checked = !!toggleAllChecked;
```

Comment:
Default props could be used to avoid bang operator

Reply:
In fact we don't even need the default props. Let's test/pass it directly